acados/dense_qp/dense_qp_qpoases.c

```diff
@@ -65,6 +65,7 @@ void dense_qp_qpoases_initialize_default_args(void *args_)
     args->max_cputime = 1000.0;
     args->warm_start = 0;
     args->max_nwsr = 1000;
+	/* args->use_precomputed_choleski = 0; */
```

Comment:
1. use_precomputed_cholesk**y** (all occurrences)
2. Should you not initialize both hotstart and use_precomputed_cholesky to zero here?

Reply:
Totally. fixed.